core/src/main/java/io/github/tcdl/msb/collector/Collector.java

```diff
@@ -42,7 +41,8 @@
 
     private int timeoutMs;
     private int currentTimeoutMs;
-    private long waitForAcksUntil;
+    private Integer waitForAcksMs;
+    private Long waitForAcksUntil;
     private int waitForResponses;
```

Comment:
ok in Requester from Node.js we use  if (this.waitForAcksMs || this.waitForResponses) ...in java version we read requestOption parameters..will do some refactoring to have both values in sync in both classes